browser/src/Editor/NeovimEditor/WelcomeBufferLayer.tsx

```diff
@@ -404,22 +507,38 @@ export class WelcomeView extends React.PureComponent<WelcomeViewProps, WelcomeVi
                         <HeroImage src="images/oni-icon-no-border.svg" />
                     </Column>
                     <Column alignment="flex-start">
-                        <SubtitleText>{`v${this.state.version}`}</SubtitleText>
+                        {version && <SubtitleText>{`v${version}`}</SubtitleText>}
```

Comment:
this change doesnt prevent the whole welcome layer from rendering if there is an issue with the `getMetadata` function not sure how it could fail but I don't think anything other than a full editor/app crash should prevent the initial welcome page from rendering